style-guide.md

```diff
@@ -0,0 +1,6 @@
+# Style Guide
+
+We strongly recommend to follow [Google's Developer Documentation Style Guide](https://developers.google.com/style) when writing developer docs except the following:
+
+- Title cases are used in capitalization in titles and headings, rather than [sentence case](https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings). Use [this website](https://titlecase.com/) to convert to proper title cases.
```

Comment:
Can we add our official product names. For example, is it `Subnet-EVM` or `Subnet EVM`?

Reply:
Yes